certified-connectors/SAS Decisioning/apiProperties.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "serverEndpoint": {
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "tabIndex": 2,
+            "required": "true"
+          },
+          "description": "Specify SAS Viya Environment URL",
+          "displayName": "SAS Viya Environment URL",
+          "tooltip": "Provide SAS Viya Environment URL"
+        }
+      },
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Authorization Token",
+          "description": "The Authorization Token for this api",
+          "tooltip": "Provide your Authorization Token",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#024f87",
+    "capabilities": [
+      "actions"
+    ],
+    "publisher": "SAS Institute, Inc.",
+    "stackOwner": "SAS Institute, Inc.",
+    "policyTemplateInstances": [
+      {
+        "templateId": "dynamichosturl",
+        "title": "Host URL",
+        "parameters": {
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('serverEndpoint')"
+        }
+      }
+    ],
+    "publisher": "SAS Institute, Inc."
```

Comment:
`publisher` is entered twice, once on line 45 and once on 34. This one can be deleted in favor of the one that's in proximity to the `stackOwner` field, since they are related properties.

Reply:
Fixed it.